### PR TITLE
Fix dead docs.ray-URL in hyperparameter tuning 

### PR DESCRIPTION
**Pull request by impaktor**
## Description
Documentation: [Hyperparameter tuning with Ray Tune](https://pytorch.org/tutorials/beginner/hyperparameter_tuning_tutorial.html) points to URL [Population Based Training](https://docs.ray.io/en/master/tune/tutorials/tune-advanced-tutorial.html) which is 404. It is supposed to look like so: [waybackmachine](https://web.archive.org/web/20200812160858/https://docs.ray.io/en/master/tune/tutorials/tune-advanced-tutorial.html). 

Replaced it with new docs: https://docs.ray.io/en/latest/tune/examples/pbt_guide.html

## Checklist
1 & 3 are not valid, this is fixing an issue that is not reported.
- [x] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [x] Only one issue is addressed in this pull request
- [x] Labels from the issue that this PR is fixing are added to this pull request
- [x] No unnessessary issues are included into this pull request.

cc @svekars @carljparker

### CODE REVIEW

**Review by svekars — APPROVED** [review]
Thanks!
